kytos/core/config.py

```diff
@@ -147,44 +149,6 @@ def parse_args(self):
 
         self.options['daemon'] = self._parse_options(argv)
 
-    @staticmethod
-    def _generate_file_from_template(templates,
-                                     destination=Path(__file__).parent,
-                                     **kwargs):
```

Comment:
It looks like this function name should be in the plural form: `_generate_files_from_templates()`, right?
My suggestion: `_render_config_templates()`

Reply:
I liked the suggestion and already applied it.